roles/taca/templates/site_taca_delivery.yml.j2

```diff
@@ -6,7 +6,7 @@ deliver:
     analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
     datapath: <ROOTPATH>/DATA/<PROJECTID>
     stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
-    deliverypath: /pica/sw/links/proj/<UPPNEXID>/INBOX/<PROJECTID>
+    deliverypath: "{{ proj_path }}}<UPPNEXID>/INBOX/<PROJECTID>"
```

Comment:
there's one `}` too many there if I'm not mistaken. since I'm nitpicking, I'd suggest to put the `/` here, rather than in the `proj_path` variable (and it won't break anything if it's in both positions)